### PR TITLE
update package.json

### PR DESCRIPTION
**Pull request by frank-dspeed**

### CODE REVIEW

**Comment by hemanth** [comment]
You can make travis happy at your localhost as well -> [trevor](http://nmotw.in/trevor/).

**Comment by frank-dspeed** [comment]
why dosn't your checks simply allow json syntax :D

**Comment by frank-dspeed** [comment]
@hemanth did you try out trevor with this repo befor there is much more failing i will open issu to make that work first

File: generators/gulp/index.js [hunk 1]
```diff
@@ -47,14 +47,15 @@ module.exports = generators.Base.extend({
 
       extend(pkg, {
         devDependencies: {
-          gulp: '^3.9.0',
+          gulp: '3.9.1',
+          'gulp-coveralls': '^0.1.4',
```
Comment (SBoudrias) [inline generators/gulp/index.js]: I think that's the wrong place to bump it. We only want to add gulp-coveralls if we're using coveralls.

**Comment by SBoudrias** [comment]
Thanks

**Comment by hemanth** [comment]
🤘

**Pull request merged**